torch/serialization.py

```diff
@@ -269,6 +270,9 @@ def _meta_tag(obj):
     if obj.device.type == 'meta':
         return 'meta'
 
+def _xpu_tag(obj):
```

Comment:
Can we unify all `_xxx_tag` to
```python
def _backend_tag(backend, obj):
    if obj.device.type == backend:
        if obj.device.index is None:
            return backend
        else:
            return backend + ':' + str(obj.device.index)
```
```

Reply:
cpu, mps and meta doesn't currently check `obj.device.index`. Does they have it always `=0`?

Reply:
Generalized cuda, hpu, xpu and privateuse1. I left cpu, mps and meta unchanged - see question above.

Reply:
Yes, you are right~